src/bin/saitocli.rs

```diff
@@ -162,6 +174,17 @@ pub async fn main() -> saito_rust::Result<()> {
                         .help("path to blocks directory"),
                 ),
         )
+        .subcommand(
+            App::new("read_tx")
+                .about("read content of the previous tx out & use the hash as next input tx")
+                .arg(
+                    Arg::with_name("filename")
+                        .short("f")
+                        .long("filename")
+                        .takes_value(true)
+                        .help("the previous tx output"),
+                ),
+        )
```

Comment:
Can we add some docs/examples to the top of this file similar to the other commands?